### ISSUE TITLE
read_file unbound keyword

### ISSUE THREAD

**Issue by TheLoneWolfling** [incrediblesound/huo#46]
Is it intentional that read_file can take an unbound keyword?

**Comment by incrediblesound** [comment]
Yes... but honestly I can't remember why I did that. I think that it is more consistent that it should take a value. I'll make that change.
